static/js/settings_linkifiers.js

```diff
@@ -127,6 +146,61 @@ export function build_page() {
         });
     });
 
+    $(".admin_linkifiers_table").on("click", ".edit", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const btn = $(this);
+        const linkifier_id = Number.parseInt(btn.attr("data-linkifier-id"), 10);
+        const modal = open_linkifier_edit_form(linkifier_id);
+
+        modal.find(".submit-linkifier-info-change").on("click", (e) => {
+            e.preventDefault();
+            e.stopPropagation();
+            const change_linkifier_button = $(".submit-linkifier-info-change");
+            change_linkifier_button.prop("disabled", true);
+
+            const url = "/json/realm/filters/" + encodeURIComponent(linkifier_id);
+            const pattern = modal.find("#edit-linkifier-pattern").val().trim();
+            const url_format_string = modal.find("#edit-linkifier-url-format-string").val().trim();
+            const data = {pattern, url_format_string};
+            const pattern_status = modal.find("#edit-linkifier-pattern-status").expectOne();
+            const format_status = modal.find("#edit-linkifier-format-status").expectOne();
+            const linkifier_status = modal.find(".edit-linkifier-status").expectOne();
+            const opts = {
+                success_continuation() {
+                    const row = btn.parents("tr");
+                    row.find(".linkifier_pattern").val(pattern);
+                    row.find(".linkifier_url_format_string").val(url_format_string);
```

Comment:
These row edit changes are unnecessary; we'll live-update the row via `server_events_dispatch` like we do other UI.  I removed them and verified the UI still works.